{"uuid": "370b45a7-125b-4858-be96-99ad346e5205", "children": ["c70b15f8-d71d-46e9-93e8-0ca294527e7a"], "befores": [{"name": "login", "status": "broken", "statusDetails": {"message": "Exception: 无该校验方式：no_check\n", "trace": "  File \"/Users/nali/anaconda3/lib/python3.7/site-packages/pluggy/callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"/Users/nali/anaconda3/lib/python3.7/site-packages/_pytest/fixtures.py\", line 964, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/Users/nali/anaconda3/lib/python3.7/site-packages/_pytest/fixtures.py\", line 792, in call_fixture_func\n    res = fixturefunc(**kwargs)\n  File \"/Users/nali/AutoApiTest/conftest.py\", line 28, in login\n    api_send_check(request.param, case_dict, case_path, relevance)\n  File \"/Users/nali/AutoApiTest/common/unit/api_send_check.py\", line 33, in api_send_check\n    check_result.check_result(test_name, check_data, code, data, case_path, relevance)\n  File \"/Users/nali/AutoApiTest/common/unit/check_result.py\", line 129, in check_result\n    raise Exception(\"无该校验方式：{}\".format(case[\"check_type\"]))\n"}, "steps": [{"name": "POST请求接口", "status": "passed", "attachments": [{"name": "请求接口：", "source": "217932ba-a3e9-4996-8d21-b7eb987af3fa-attachment.attach"}, {"name": "用例描述：", "source": "85d36acd-1497-4f8f-86a6-618c4c616e85-attachment.attach"}, {"name": "请求地址", "source": "44a5edd6-5e3a-42fb-b14a-c02c2816e371-attachment.attach"}, {"name": "请求头", "source": "0d9102c2-b2db-4714-9293-da2cbd75a5ad-attachment.attach"}, {"name": "请求参数", "source": "7e11ecb4-0a3e-44b6-b768-7e11cb5d7d84-attachment.attach"}], "start": 1617514045983, "stop": 1617514045985}, {"name": "不校验结果", "status": "passed", "start": 1617514046352, "stop": 1617514046352}], "start": 1617514045980, "stop": 1617514046353}], "start": 1617514045980, "stop": 1617514046415}